src/attribute.ts

```diff
@@ -199,6 +233,21 @@ export class Attribute {
     this.items = this.items.filter(i => i.index! !== item.index!)
   }
 
+  // addExtension finds the "extension" child of an attribute (if any)
+  // and add an item to this array attribute. The child has a custom definitionId
+  // which is a profile on the Extension type.
+  addExtension(definitionId: string, index?: number) {
+    const childExtension = this.children.find(c => c.isExtension)
+    if (!childExtension)
+      throw new Error(`attribute ${this.path} has not extension child`)
```

Comment:
typo: has no extension child?

Reply:
this error is thrown when an attribute does not have a child of type `Extension`. Do you want me to change the error message ?

Reply:
I would have said "attribute ${this.path} has no extension child" instead of "attribute ${this.path} has not extension child"